components/sections/plugins/index.js

```diff
@@ -231,8 +131,7 @@ export default () => (
           display: block;
           overflow-x: scroll;
           -webkit-overflow-scrolling: touch;
-
-          padding-bottom: 48px;
+          padding-top: 10px;
```

Comment:
This is because of a weird behavior where because of the `overflow-x: scroll`, vertical overflow was also hidden and on hover, the top part of the plugin div was being cut off

Reply:
yeah, weird